### ISSUE TITLE
Warning treated as error in "glPaintTest.cpp".

### ISSUE THREAD

**Issue by KelSolaar** [PixarAnimationStudios/OpenSubdiv#1156]
I'm having issues compiling because of a warning emitted in *glPaintTest.cpp*:

```
D:\OpenSubdiv\OpenSubdiv\examples\glPaintTest\glPaintTest.cpp(992): error C2220: warning treated as error - no 'object' file generated [D:\OpenSubdiv\build\platform-windows\arch-AMD64\python-2.7.14\opensubdi
v\examples\glPaintTest\glPaintTest.vcxproj]
D:\OpenSubdiv\OpenSubdiv\examples\glPaintTest\glPaintTest.cpp(992): warning C4244: 'argument': conversion from 'int' to 'float', possible loss of data [D:\OpenSubdiv\build\platform-windows\arch-AMD64\python-
2.7.14\opensubdiv\examples\glPaintTest\glPaintTest.vcxproj]
```

I'm using the 3.4.0 tagged release.

Cheers,

Thomas

**Comment by barfowl** [comment]
Thanks, Thomas.  This points out the fact that the OpneGL examples are not vetted with MSVC as part of our automated integration testing -- something we should address.  I have the required change made but cannot build with MSVC locally, so a pull request will be delayed until I can verify it (and even then, it will remain in the dev branch once merged).

In the meantime, unless you really need the examples, you can build with NO_EXAMPLES=1 to avoid this.

**Comment by KelSolaar** [comment]
Hi Barry (@barfowl),

Sorry I should have said it was on Windows with MSVC (even if obvious). I excluded the examples from the build and it is all good!

Cheers,

Thomas

**Comment by jilliene** [comment]
Filed as internal issue #OSD-300

**Comment by barfowl** [comment]
This should now be addressed with the merger of pull request #1158.

We're also making progress of replacing the Appveyor integration tests with something that will allow us to build both OpenGL and DirectX examples with MSVC -- more to come on that in the coming weeks.

Thanks again for bringing this to our attention.
